README.mdown

```diff
@@ -54,6 +54,7 @@ If you want to use secure mode, ensure you set your API secret in `config/initia
 ```ruby
   config.api_secret = '123456'
 ```
+**Note: This example is just for the sake of simplicity, you should never include your api secret in source control, you should instead use the rails [secret config](http://guides.rubyonrails.org/4_1_release_notes.html#config-secrets-yml) feature.**
```

Comment:
Going to be a grammar nerd here:

> ...api secret in source control, you should instead use the rails...

is kind of a run on sentence. It should be split up into two sentences: "...api secret in source control. You should instead use the rails...".